synapseclient/extensions/curator/file_based_metadata_task.py

```diff
@@ -51,16 +53,17 @@ def create_json_schema_entity_view(
     Returns:
         The Synapse id of the crated entity view
     """
-    js_service = syn.service("json_schema")
-    json_schema = js_service.get_json_schema(synapse_entity_id)
-    org = js_service.JsonSchemaOrganization(
-        json_schema["jsonSchemaVersionInfo"]["organizationName"]
+    entity = get(
+        file_options=FileOptions(download_file=False),
+        synapse_id=synapse_entity_id,
+        synapse_client=syn,
     )
-    schema_version = JsonSchemaVersion.from_response(
-        org,
-        json_schema["jsonSchemaVersionInfo"],
-    )
-    columns = _create_columns_from_json_schema(schema_version.body)
+    assert isinstance(entity, (Folder, Project))
```

Comment:
@BryanFauble This is failing a unit test because the entity is patched with a `Mock` class. Should I add Mock here, or is there a better workaround?